must-gather/collection-scripts/gather_clusterscoped_resources

```diff
@@ -72,3 +72,23 @@ for command_desc in "${commands_desc[@]}"; do
      COMMAND_OUTPUT_FILE=${BASE_COLLECTION_PATH}/cluster-scoped-resources/oc_output/desc_${command_desc// /_}
     { oc describe "${command_desc}"; } >> "${COMMAND_OUTPUT_FILE}"
 done
+
+# Collect logs for drpolicy and mirrorpeers
```

Comment:
(nit) comment should read collection of resources and not logs (in line 79 as well below)